packages/mobile/src/services/offline-downloader/offline-downloader.ts

```diff
@@ -335,9 +353,11 @@ export const downloadTrack = async (trackForDownload: TrackForDownload) => {
       return
     }
 
-    await downloadTrackCoverArt(trackFromApi)
+    await Promise.all([
```

Comment:
This makes sense for speed, but does it mean we have more goin on in background? Prob not a big deal either way?

Reply:
js & react native should be really good at handling this, lots of idle IO time, especially as the art will come back before the track in 99% of cases so the parsing of the response happens while the other request is just idling